### DISCUSSION TITLE
v1.6.3

### DISCUSSION THREAD

**Discussion by SagalVanderklok** [SagalVanderklok/ChatRoomRecorder#39] [Announcements]
### Changelog
---
- Implement chat room URLs copying
- Implement bulk action and resolution changing
- Implement opening and deleteng files using a keyboard
- Remove a filter after adding a chat room
- Fix the bug with the selection after deleting and moving chat rooms
- Fix the tab order of the form controls 
- Use images for the embedded web brower navigation buttons

<hr /><em>This discussion was created from the release <a href='https://github.com/SagalVanderklok/ChatRoomRecorder/releases/tag/v1.6.3'>v1.6.3</a>.</em>

**Comment by Twili86** [comment]
Could you add the ability to record from the Chaturbate mirror?  "chaturbate.md"

**Comment by zxczxc322** [comment]
I noticed some stripchat recordings began to freezes at a certain time 
If you stop recording by manually or streamer goes offline, file has size of full recording and after converting to mp4 has correct timings, but with lags after time at recording was freeze in ts file.
I guess this is due lags on the broadcast itself and recording not interrupted for some reason and  just continues to record with errors

**Comment by dwa99** [comment]
Works like a charm, brilliant soft, thank you!
